pkg/pgmodel/ingestor/ingestor.go

```diff
@@ -93,19 +93,26 @@ func (ingestor *DBIngestor) Ingest(ctx context.Context, r *prompb.WriteRequest)
 	var (
 		timeseries = r.Timeseries
 		metadata   = r.Metadata
+		size       = r.Size()
 	)
 	release := func() { FinishWriteRequest(r) }
 	switch numTs, numMeta := len(timeseries), len(metadata); {
 	case numTs > 0 && numMeta == 0:
 		// Write request contains only time-series.
 		n, err := ingestor.ingestTimeseries(ctx, timeseries, release)
+		if err != nil {
+			regMetricIngestorBytes(size)
```

Comment:
Why are we registering size of requests that failed to ingest when the aim is to track `ingested` part. Failed ones are never ingested.

Reply:
ops this is obviously an error :)

Reply:
Plus size of failed requests are generally off as those requests are malformed.

Reply:
My goal was to register only successful ones but turned logic upside down :)